h/static/scripts/annotation-metadata.js

```diff
@@ -70,30 +70,55 @@ function isNew(annotation) {
   return !annotation.id;
 }
 
+/** Return `true` if the given annotation is a page note, `false` otherwise. */
+function isPageNote(annotation) {
+  if (annotation && !annotation.deleted) {
```

Comment:
Annotations do not have a `deleted` property. See this discussion on Slack: https://hypothes-is.slack.com/archives/public/p1466767221002927